src/OnlineSales/Interfaces/IEmailWithLogService.cs

```diff
@@ -0,0 +1,12 @@
+﻿// <copyright file="IEmailWithLogService.cs" company="WavePoint Co. Ltd.">
+// Licensed under the MIT license. See LICENSE file in the samples root for full license information.
+// </copyright>
+
+namespace OnlineSales.Interfaces;
+
+public interface IEmailWithLogService
```

Comment:
@gsplakmal we need support for templateId parameter here.

Reply:
Updated.